tests/internal/config/default.yaml

```diff
@@ -11,7 +11,7 @@ control_plane_label: "control-plane"
 tc_prefix: "TC-"
 mco_namespace: "openshift-machine-config-operator"
 mco_config_daemon_name: "machine-config-daemon"
+logging_namespace: "openshift-logging"
```

Comment:
Please re-name to logging_operator_namespace

Reply:
Fixed